app/build.gradle

```diff
@@ -120,7 +120,7 @@ dependencies {
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
 
     // Tor
-    implementation "org.briarproject:onionwrapper-android:0.0.4"
+    implementation "org.briarproject:onionwrapper-android:0.0.5"
     tor 'org.briarproject:tor-android:0.4.7.13-2'
```

Comment:
Can/should we update tor to 0.4.7.14 at the same time?

Reply:
Ah looks like we haven't released that yet for some reason. Considering the onion services issue, we probably should.

Reply:
Yup, there are some extra Docker hoops to jump through for the Mac binaries but I should be able to upload them tomorrow. I'll leave this open till then and add a commit to upgrade the Tor binaries.